code/localization/localize.h

```diff
@@ -37,6 +37,7 @@
 // language info table
 typedef struct lang_info {
 	char lang_name[LCL_LANG_NAME_LEN + 1];				// literal name of the language
+	int xstr;											// the xstr id for the language name itself. Used for the built-in langauge selector
```

Comment:
More of a nitpick. Language is misspelled as "langauge" in the later part of this string.

Reply:
@JohnAFernandez promised I got all those typos! 😛